pallets/fragments/src/mock.rs

```diff
@@ -109,32 +116,40 @@ where
 impl pallet_randomness_collective_flip::Config for Test {}
 
 impl pallet_balances::Config for Test {
-	type Balance = u64;
+	type Balance = Balance;
 	type DustRemoval = ();
 	type Event = Event;
-	type ExistentialDeposit = ConstU64<1>;
+	type ExistentialDeposit = ConstU128<1>;
 	type AccountStore = System;
 	type WeightInfo = ();
 	type MaxLocks = ();
 	type MaxReserves = ();
 	type ReserveIdentifier = [u8; 8];
 }
 
+parameter_types! {
```

Comment:
wow lot of changes... is that cos of the min balance testing?

Reply:
yup. but since we reverted back to `deposit_creating` (and `pallet_balances`) - it makes no difference 😅.

but yup, as you see in mock.rs - previously we set the associated type for `Currency` as `()` 🤦‍♀️. That's why we couldn't catch the error you had when running it as a local node